platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigratable.java

```diff
@@ -152,13 +159,14 @@ public void doVersionUpgradeImport(ImportMigrationContext context) {
   }
 
   private boolean isAcceptedConfigurationAdminEntry(ImportMigrationConfigurationAdminEntry entry) {
-    if (ACCEPTED_ENTRY_PIDS.contains(entry.getPid())) {
-      return true;
-    }
-    if (ACCEPTED_ENTRY_FACTORY_PIDS.contains(entry.getFactoryPid())) {
-      return true;
-    }
-    return false;
+    return (ACCEPTED_ENTRY_PIDS.contains(entry.getPid())
+        || ACCEPTED_ENTRY_FACTORY_PIDS.contains(entry.getFactoryPid())
+        || isASource(entry.getFactoryPid()));
+  }
+
+  private boolean isASource(String factoryPid) {
+    return (StringUtils.containsIgnoreCase(factoryPid, "source")
```

Comment:
❓ Isn't these two tests very generic? Should we be testing for more to avoid false positive?

Reply:
This is being handled similarly to this https://github.com/codice/ddf/blob/60e14950eaf64b0a2e95d112133fb26b2a4153d7/catalog/spatial/registry/registry-source-configuration-handler/src/main/java/org/codice/ddf/registry/source/configuration/SourceConfigurationHandler.java#L83 in order to make sure downstream projects are getting sources as well

Reply:
@paouelle I agree that this is a somewhat brittle way to handle this in the system, but this is how the CSW Registry has handled it for a long time and is a stable capability.

Reply:
Can we add a comment that explains why and reference the code we are trying to mimic so the next guy won't wonder?